src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -26,17 +26,25 @@ If you're updating your Sentry SDK to the latest version, check out our [migrati
 
 Create three files in the root directory of your project, `sentry.client.config.js`, `sentry.server.config.js` and `sentry.edge.config.js`. In these files, add your initialization code for the client-side SDK and server-side SDK, respectively. We've included some examples below.
 
-For each configuration:
+The three configuration types are mostly the same, except that some configuration, like the one for Session Replay, only works in `sentry.client.config.js`.
```

Comment:
```suggestion
Please note that there are slight differences between these files since they run in different places (browser, server, edge), so copy them carefully!
```

I would word it the other way round. Instead of saying "they're _mostly_ the same" I would say "there are differences!".

Otherwise what we're saying here makes sense.